### ISSUE TITLE
GLTFLoader's aoMap uv not match Blender exported and other GLTFViewers.

### ISSUE THREAD

**Issue by gonnavis** [mrdoob/three.js#21641]
**Describe the bug**

This glb file [sofa.zip](https://github.com/mrdoob/three.js/files/6301192/sofa.zip) which made and exported from Blender, has diffuseMap and aoMap.

When loaded to threejs scene using GLTFLoader, the aoMap display incorret.
![image](https://user-images.githubusercontent.com/10785634/114496195-ffa1a600-9c51-11eb-91fc-e6340ecfcc63.png)

I guess they has different uvs and tried to add this line `sofa.geometry.attributes.uv2.copy(sofa.geometry.attributes.uv)` then display correct.
![image](https://user-images.githubusercontent.com/10785634/114496293-37a8e900-9c52-11eb-92b1-4b3490088877.png)

I also tried https://threejs.org/editor/ and https://modelviewer.dev/editor/ ( which I heard also based on threejs ), all has the same problem.
![image](https://user-images.githubusercontent.com/10785634/114496645-febd4400-9c52-11eb-8109-f771fb6f410c.png)
![image](https://user-images.githubusercontent.com/10785634/114496653-05e45200-9c53-11eb-9075-c6b8eef8a3aa.png)

But with https://sandbox.babylonjs.com/ , https://playcanvas.com/viewer and Win10 3D Viewer, display all ok.
![image](https://user-images.githubusercontent.com/10785634/114496775-40e68580-9c53-11eb-9411-8d081b3e655a.png)
![image](https://user-images.githubusercontent.com/10785634/114496785-45ab3980-9c53-11eb-936b-1853bdd8f1eb.png)
![image](https://user-images.githubusercontent.com/10785634/114496795-4ba11a80-9c53-11eb-9674-843ec1605fe9.png)

Is the problem of threejs's GLTFLoader?

**To Reproduce**

1. Go to https://threejs.org/editor/
2. Click on File -> Import, chose the glb file, and add a AmbientLight
3. See error

***Code***

```js
let loader=new GLTFLoader()
loader.load('./sofa.glb',gltf=>{
  scene.add(gltf.scene)
  let sofa=gltf.scene.children[0]
  // sofa.geometry.attributes.uv2.copy(sofa.geometry.attributes.uv)
})
```

***Live example***

See above metioned gltf viewer and load the glb model.

**Expected behavior**

Display correct aoMap with this glb model like in https://sandbox.babylonjs.com/ , https://playcanvas.com/viewer and Win10 3D Viewer.

**Screenshots**

See above.

**Platform:**

 - Device: [Desktop]
 - OS: [Windows]
 - Browser: [Chrome]
 - Three.js version: [r127]

**Comment by donmccurdy** [comment]
While the glTF specification allows any material texture slot to use any UV channel, three.js requires most material texture slots to use UV1, and requires AO textures or lightmaps to use UV2. If the model only has one UV channel, GLTFLoader will automatically duplicate it if needed for an AO texture:

https://github.com/mrdoob/three.js/blob/6bed409398dd67d6d2d887e3c19efcf26f185eb9/examples/js/loaders/GLTFLoader.js#L2503-L2507

However, it skips that if doing so would overwrite an existing UV channel, and in this case the model already has a second (unused) UV channel. Potentially that could be fixed, but there are a lot of messy edge cases involved in doing so.

The easier workaround is to remove the unused UV channel in Blender. The better, long-term solution is https://github.com/mrdoob/three.js/issues/12608.

**Comment by Mugen87** [comment]
In this case, the issue can be merged in #12608.
